integration/metrics_test.go

```diff
@@ -114,10 +114,28 @@ func TestOverlayFallbackMetric(t *testing.T) {
 		{
 			name:  "image with fs.Mount errors results in non-zero overlay fallback",
 			image: rabbitmqImage,
+			indexDigestFn: func(sh *shell.Shell, contentStoreType store.ContentStoreType, image imageInfo) string {
+				corruptZtocDigest := "b6f44b39963e56996b1273cf1c686089e77787f9cebe191b6c8eb7ef9361e625"
```

Comment:
Where did this digest come from? Can we choose a random ztoc digest from the index instead?

Reply:
It was from the rabbitmq image. Yeah, we can just generate it dynamically.